lib/provision.rb

```diff
@@ -62,12 +72,13 @@ def install_squid
     def configure_apt
       executer = Executer.new("data/setup")
       # send apt through squid for caching
+      executer.run_in_vm!("sudo touch /etc/apt/apt.conf.d/99http-proxy")
```

Comment:
This should be unnecessary; all it does is create the file which will be created on the next line anyway...